### PR TITLE
Bug: Fix command error messages

### LINKED ISSUE #367: Bug: Find command error message showing incorrect parameters and description
image

* Parameters for weight and height should be `RANGE`
* Description "Lists all clients whose specified attribute **contains the specified keyword** does not apply for weight and height filters

### PR DESCRIPTION
**Pull request by RyanNgWH**
Fixes #367.

Also fixes some minor errors in the messages of other commands

### CODE REVIEW

**Review by dillontkh — APPROVED** [review]
LGTM!

**Review by ruijietay — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/messages/FitDeleteCommandMessages.java [hunk 1]
```diff
@@ -10,27 +10,28 @@ public class FitDeleteCommandMessages extends Messages {
...
-        + "Parameters: INDEX (must be a positive integer) "
-        + PREFIX_EXERCISE_NAME + "EXERCISE_NAME\n"
-        + "Example: " + COMMAND_WORD + " 1 "
-        + PREFIX_EXERCISE_NAME + "squats\n"
-        + "Parameters: INDEX (must be a positive integer) "
-        + PREFIX_FITDELETE_DELETE_ALL + "\n"
-        + "Example: " + COMMAND_WORD + " 1 "
-        + PREFIX_FITDELETE_DELETE_ALL;
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the specified exercise from the client identified "
```
Comment (jjchee77) [inline src/main/java/seedu/address/logic/messages/FitDeleteCommandMessages.java:14]: "Deletes the exercise specified by name, from the client...." should be clearer.

**Review by jjchee77 — APPROVED** [review]
Just one nit to pick.

**Pull request merged**